mwui-angular/README.md

```diff
@@ -2,15 +2,23 @@
 
 ## MWUI - Angular Integration
 
-Work in Progress
-
-### Required fixes
-
 #### Angular
 
-To enable support for web components in Angular, add the following to the `@NgModule` in `app.module.ts`:
+To use the Design-System import `MwuiComponentLibraryModule` in `app.module.ts`:
```

Comment:
picky, but let's refer to "component library" instead of Design-System (as this is a term for the whole thing)